packages/valory/skills/apy_estimation_abci/tests/test_behaviours.py

```diff
@@ -277,6 +278,7 @@ def test_current_pair_ids(
         behaviour.params.pair_ids = pairs_ids
 
         assert expected_pair_ids == behaviour.current_pair_ids
+        self.end_round()
```

Comment:
Ending round after all tests.

Reply:
you can also do this in `teardown` next time. Would be good to collect several refactoring opportunities at once though.